source/atlas-cli-changelog.txt

```diff
@@ -56,6 +60,20 @@ Released 31 May 2023
 
 Released 25 May 2023
 
+- Introduces a breaking change to the following commands
+  that modifies their ``--output json`` response structures:
+
+  - :ref:`atlas-teams-list`
+  - :ref:`atlas-dbusers-certs-list`
+  - :ref:`atlas-dbusers-list`
+  - :ref:`atlas-projects-apiKeys-list`
+  - :ref:`atlas-organizations-apiKeys-list`
+  - :ref:`atlas-projects-users-list`
+  - :ref:`atlas-processes-list`
+  
+  Use the nested ``.results`` object to access the previous response
+  structure.
```

Comment:
Optional: additional lines for script migration suggested approach - not sure if there's a precedent for this in our docs or the previous sentence is sufficient
```suggestion
  structure.
  To migrate your script we recommend: ``jq '.' ...`` => ``jq '.results' ...``
```

Reply:
@matt-condon added! when you have a second can you please take a look to ensure the copy and the template looks good?